pygfx/objects/_skins.py

```diff
@@ -0,0 +1,206 @@
+import numpy as np
+import pylinalg as la
+from typing import List
+from ._base import WorldObject
+from ..utils import array_from_shadertype
+from ..utils.transform import AffineBase, callback
+from ..utils.enums import BindMode
+from ..resources import Buffer
+from ._more import Mesh
+
+
+class Bone(WorldObject):
+    """A bone.
+
+    A bone is a part of a skeleton.The skeleton in turn is used by the SkinnedMesh.
+    Bones are almost identical to a blank WorldObject.
```

Comment:
Do these Bone objects *contain* `Mesh` objects? And if so, can they contain more than one mesh?

Reply:
No.
Bone objects are almost just blank 3D objects, invisible and not actually rendered. They only exist to manage and maintain world transforms

Reply:
Sorry, I meant are the actual meshes children of the `Bone` objects?

Reply:
No.
The bone objects and the actual Mesh are independent. 
Through the `bind` method of `SkinedMesh`, Skeleton can be bound to a specific Mesh, thereby driving the animation of the Mesh. 
The same skeleton can be bound to different Meshes, and theoretically, a Mesh can also be bound to different skeletons.

Reply:
🤯 thanks for explaining!